applications/data_analysis/general_applied_math.ipynb

```diff
@@ -875,6 +876,71 @@
     "print(f\"\\t\\tlog (np.log)     = {np.log(input_list)}\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sorting\n",
+    "\n",
+    "Python includes functions to organize and sort lists and arrays"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### `sort`\n",
+    "\n",
+    "Sort numerical and alphanumeric strings in arrays\n",
+    "\n",
+    "`sorted()` is both part of the standard Python library and can be calculated with `numpy`\n",
+    "\n",
+    "Python has two similar sort function, `sort()` and `sorted()`. `sort()` will reorganize the list in-place and return `None`, while `sorted()` creates a new copy with the sorted elements.\n",
```

Comment:
```suggestion
    "Python has two similar sorting functions, `list.sort()` and `sorted()`. `list.sort()` is a method available for lists that will reorganize a list in-place and return `None`, while `sorted()` is a function that creates a new sorted list from the inputs.\n",
```

I'm not committed to this suggestion, but I think it's a little more consistent with the Python docs.  

How to describe / indicate methods in this resource will probably come up again as well.